src/geo/projection/globe_util.js

```diff
@@ -68,6 +67,10 @@ const GLOBE_LOW_ZOOM_TILE_AABBS = [
     new Aabb([0, 0, GLOBE_MIN], [GLOBE_MAX, GLOBE_MAX, GLOBE_MAX])  // x=1, y=1
 ];
 
+export function globeMetersToEcef(d: number): number {
+    return d * mercatorZfromAltitude(1, 0.0) * 2.0 * GLOBE_RADIUS * Math.PI;
```

Comment:
Nit: we could simplify this. `mercatorZfromAltitude(1, 0) = 1 / earthCircumference = 1 / 2 * Math.PI * earthRadius`, so we would have:

```
return d * GLOBE_RADIUS / earthRadius;
```